src/utils/constants.ts

```diff
@@ -247,7 +247,7 @@ export const institutionType4Product = (productId: string | undefined) => {
         (it) =>
           it.labelKey === 'pa' ||
           it.labelKey === 'gsp' ||
-          (ENV.ENV !== 'PROD' && it.labelKey === 'psp') ||
+          (ENV.PRV.SHOW && it.labelKey === 'psp') ||
```

Comment:
Will PSP's have to be available also in production when we enable private in production?

Reply:
regression